blog/2024-06-28-data-flow-with-vibration-data/index.mdx

```diff
@@ -0,0 +1,132 @@
+
+### Advantages of Storing Raw Data
+Storing raw data in the context of vibration monitoring offers significant advantages. Raw sensor output stored as a blob captures the full fidelity of the original signal, allowing extensive post-processing and re-analysis using different algorithms or filters.
+
+This flexibility is essential for developing new diagnostic tools or improving existing ones without the need for repeated data acquisition. For example, raw data can be used to perform detailed frequency analysis using FFT (see example below), detect spikes using time domain analysis, identify signal variations using wavelet transform, find faults in rotating equipment using envelope analysis, and understand structural vibrations using modal analysis.
+
+![FFT Image Example](./img/fft_example.webp)
+
+In this example, the Fast Fourier Transform (FFT) is calculated to identify the frequency content of the signal. 
+This information can be used to identify the dominant frequencies in the signal, which in turn can be an indication of rotating equipment faults such as unbalance, misalignment or bearing failure.
+
+## Time Series Databases
+Time Series Databases (TSDBs) are specialised data storage systems optimised for handling time-indexed data. They excel at managing large volumes of sequentially generated data points, such as vibration measurements, by providing efficient write and read operations.
+
+### Time Series Object Store vs. Traditional Time Series Database
+A time series object store and a traditional time series database can serve similar purposes, but have distinct architectural differences. While a traditional TSDB is optimised for high-speed ingestion and retrieval of scalar data points, an object store is designed to handle complex, high-dimensional data objects and their metadata. This makes the object store more versatile for applications that require rich contextual information, such as vibration analysis that includes waveform data and diagnostic logs.
+
+### Data Flow With a Time Series Object Store
+With a time series object store, each chunk of vibration data is stored as a binary object with a timestamp along with metadata such as preprocess metrics that can be useful for filtering and replication (more on that later).
+
+![Data Flow Schema](./img/vibration_data_flow.webp)
+
+This method allows efficient management of large vibration datasets by providing fast access to specific time periods. In fact, [**ReductStore outperforms TimescaleDB for blobs ranging from 100KB and larger**](<https://www.reduct.store/blog/comparisons/iot/reductstore-vs-timescaledb>) using this method, with improvements ranging from 205% to 1300%.
+
+## Eliminate Data Loss
```

Comment:
This title confuses me a bit. We can't avoid data lost completely. I would say something like "Care for Data Reduction"